### PR TITLE
Add Dataloader for MIT Adobe 5K dataset

### PR DESCRIPTION
**Pull request by soumik12345**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ on:
       - mirnetv2/**
       - tests.py
       - requirements.txt
+env:
+  WANDB_API_KEY: ${{ secrets.WANDB_API_KEY }}
```
Comment (SauravMaheshkar) [inline .github/workflows/tests.yml:10]: Why do we need an API key for the tests ? All the tests have `visualize_on_wandb` set to `False` ?

Comment (soumik12345) [reply .github/workflows/tests.yml:10]: @SauravMaheshkar We need the API key for fetching the artifacts from wandb.

**Comment by soumik12345** [comment]
@SauravMaheshkar can you rebase this branch once to avoid any potential conflicts with https://github.com/soumik12345/mirnetv2/pull/8 which has been merged?

File: .github/workflows/tests.yml (hunk 1 above)
Comment (SauravMaheshkar) [reply .github/workflows/tests.yml:10]: Ahh yes ofcourse got it 👍🏽

**Review by SauravMaheshkar — APPROVED** [review]

**Comment by SauravMaheshkar** [comment]
Ahh wait a sec, messed up the `tests.yml`

**Comment by SauravMaheshkar** [comment]
@soumik12345 Now it should be okay.

**Pull request merged**